src/Workspaces/VisualBasic/Portable/VBWorkspaceResources.resx

```diff
@@ -167,24 +171,30 @@
   </data>
   <data name="Introduces_a_type_conversion_operation_similar_to_CType_The_difference_is_that_CType_succeeds_as_long_as_there_is_a_valid_conversion_whereas_DirectCast_requires_that_one_type_inherit_from_or_implement_the_other_type" xml:space="preserve">
     <value>Introduces a type conversion operation similar to CType. The difference is that CType succeeds as long as there is a valid conversion, whereas DirectCast requires that one type inherit from or implement the other type.</value>
+    <comment>{Locked="CType"}{Locked="DirectCast"}</comment>
   </data>
   <data name="The_type_name_to_return_a_System_Type_object_for" xml:space="preserve">
     <value>The type name to return a System.Type object for.</value>
+    <comment>{Locked="System.Type"}</comment>
   </data>
   <data name="Returns_a_System_Type_object_for_the_specified_type_name" xml:space="preserve">
     <value>Returns a System.Type object for the specified type name.</value>
+    <comment>{Locked="System.Type"}</comment>
   </data>
   <data name="The_XML_namespace_prefix_to_return_a_System_Xml_Linq_XNamespace_object_for_If_this_is_omitted_the_object_for_the_default_XML_namespace_is_returned" xml:space="preserve">
     <value>The XML namespace prefix to return a System.Xml.Linq.XNamespace object for. If this is omitted, the object for the default XML namespace is returned.</value>
+    <comment>{Locked="System.Xml.Linq.XNamespace"}</comment>
```

Comment:
Also lock XML or is that obvious?

Reply:
It seems if XML doesn't get translated, it's because XML has the same form in both languages. 🤷‍♂️